[BUG] linux preflight check unit tests are failing on `go1.21.6`

currently running `make test` from `main` branch on a linux platform with `go1.21.6` the following errors are reported:
```
=== RUN   TestCountPreflights
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:540
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x2672120
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func2
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:541
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x2671f00
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func3
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:542
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x2671ce0
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func4
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:544
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x2671ac0
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func5
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:545
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x26718a0
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func6
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:546
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x2671680
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func7
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:548
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x2671460
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func8
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:549
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x2671240
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func9
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:550
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x2671020
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func10
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:552
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x2670e00
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func11
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:553
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x2670be0
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func12
    preflight_linux_test.go:508:
                Error Trace:    /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:508
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:528
                                                        /home/anjan/Work/github.com/code-ready/main/pkg/crc/preflight/preflight_linux_test.go:554
                Error:          Not equal:
                                expected: 0x2659080
                                actual  : 0x26709c0
                Test:           TestCountPreflights
                Messages:       github.com/crc-org/crc/v2/pkg/crc/preflight.bundleCheck.checkBundleExtracted.func1 != github.com/crc-org/crc/v2/pkg/crc/preflight.init.checkBundleExtracted.func13
--- FAIL: TestCountPreflights (0.01s)

```

--- a/pkg/crc/preflight/preflight_linux_test.go
+++ b/pkg/crc/preflight/preflight_linux_test.go
@@ -95,7 +95,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkCrcNetworkManagerDispatcherFile},
 			{check: checkLibvirtCrcNetworkAvailable},
 			{check: checkLibvirtCrcNetworkActive},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -133,7 +133,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkCrcDnsmasqConfigFile},
 			{check: checkLibvirtCrcNetworkAvailable},
 			{check: checkLibvirtCrcNetworkActive},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -165,7 +165,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkDaemonSystemdService},
 			{check: checkDaemonSystemdSockets},
 			{check: checkVsock},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -204,7 +204,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkCrcNetworkManagerDispatcherFile},
 			{check: checkLibvirtCrcNetworkAvailable},
 			{check: checkLibvirtCrcNetworkActive},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -242,7 +242,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkCrcDnsmasqConfigFile},
 			{check: checkLibvirtCrcNetworkAvailable},
 			{check: checkLibvirtCrcNetworkActive},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -274,7 +274,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkDaemonSystemdService},
 			{check: checkDaemonSystemdSockets},
 			{check: checkVsock},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -313,7 +313,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkCrcNetworkManagerDispatcherFile},
 			{check: checkLibvirtCrcNetworkAvailable},
 			{check: checkLibvirtCrcNetworkActive},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -351,7 +351,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkCrcDnsmasqConfigFile},
 			{check: checkLibvirtCrcNetworkAvailable},
 			{check: checkLibvirtCrcNetworkActive},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -383,7 +383,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkDaemonSystemdService},
 			{check: checkDaemonSystemdSockets},
 			{check: checkVsock},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -423,7 +423,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkCrcNetworkManagerDispatcherFile},
 			{check: checkLibvirtCrcNetworkAvailable},
 			{check: checkLibvirtCrcNetworkActive},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -462,7 +462,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkCrcDnsmasqConfigFile},
 			{check: checkLibvirtCrcNetworkAvailable},
 			{check: checkLibvirtCrcNetworkActive},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 	{
@@ -495,7 +495,7 @@ var checkListForDistros = []checkListForDistro{
 			{check: checkDaemonSystemdSockets},
 			{configKeySuffix: "check-apparmor-profile-setup"},
 			{check: checkVsock},
-			{check: checkBundleExtracted(constants.GetDefaultBundlePath(preset.OpenShift))},
+			{configKeySuffix: "check-bundle-extracted"},
 		},
 	},
 }

